app/src/main/java/at/bitfire/icsdroid/ui/EditCalendarActivity.kt

```diff
@@ -246,7 +242,7 @@ class EditCalendarActivity: AppCompatActivity() {
          */
         fun updateSubscription(
             subscriptionSettingsModel: SubscriptionSettingsFragment.SubscriptionSettingsModel,
-            credentialsModel: CredentialsModel
+            credentialsModel: CredentialsFragment.CredentialsModel
```

Comment:
I think this should be just `CredentialsModel`

Reply:
Ah yes, overlooked that one